pulp.spec

```diff
@@ -390,6 +390,8 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_var}/lib/%{name}/uploads
 %dir %{_var}/log/%{name}
 %{_var}/www/pub
+%{_var}/cache/%{name}/
```

Comment:
Do we want to chown this to apache? It seems right, but what does Pulp write into this directory?